core/templates/catalog/checked-out.html

```diff
@@ -26,8 +26,8 @@
         <td><%= item.name %></td>
         <td><%= item.owner %></td>
         <td data-date="<%=item.due.getTime()%>"><%= moment(item.due).format("MMM D") %></td>
-        <td><input type=button value='Extend'></td>
-        <td><input type=button value='Check In'></td>
+        <td><input type=button value='Extend' id="btn" class="mainBtn"></td>
+        <td><input type=button value='Check In' id="btn" class="mainBtn"></td>
```

Comment:
What would happen if you switched `id="btn"` to something like a class? The HTML5 specification, unfortunately, doesn't allow for multiple IDs, so this might cause some problems accessing these elements with JavaScript down the road